### PR TITLE
Update docs for v1.1.1

### PR DESCRIPTION
**Pull request by keithmattix**
Awaiting #361 before merging

### CODE REVIEW

**Review by nojnhuh — DISMISSED** [review]
I know the docs say to open a PR with just these changes, but let's include any changes from the "3. Update API reference documentation" and "4. Update error code documentation" steps from the [release guide](https://github.com/openservicemesh/osm/blob/main/docs/release_guide.md) too (ideally `--amend`-ing this last commit bumping the version with those changes so we can do a plain merge to keep the cherry-picks separate and all the version-related updates together).

File: config.toml [hunk 1]
```diff
@@ -83,7 +83,7 @@ project_home = "https://openservicemesh.io/"
 
 # Version params
 osm_branch = "release-v1.1"
-osm_version = "v1.1.0"
+osm_version = "v1.1.1"
```
[the remark this answers is not in the document]
Comment (keithmattix) [reply config.toml]: Makes sense, done!

**Review by nojnhuh — COMMENTED** [review]
LGTM pending #361

**Comment by nojnhuh** [comment]
This commit wasn't backported in the osm repo so let's not backport it here: https://github.com/openservicemesh/osm-docs/pull/364/commits/aa3a4c7588ca35f6281de431d325346ce64ef974

**Pull request merged**